knesset_data/dataservice/bills.py

```diff
@@ -0,0 +1,39 @@
+from base import (
+    BaseKnessetDataServiceCollectionObject, BaseKnessetDataServiceFunctionObject,
+    KnessetDataServiceSimpleField, KnessetDataServiceLambdaField
+)
+import constants
+
+
+class Bill(BaseKnessetDataServiceCollectionObject):
+    SERVICE_NAME = "api"
+    METHOD_NAME = "KNS_Bill"
+    ORDERED_FIELDS = [
+        ("id", KnessetDataServiceSimpleField('BillID', "integer", "the primary key")),
+        ("kns_num", KnessetDataServiceSimpleField('KnessetNum', "integer", "kneset number")),
+        ("name", KnessetDataServiceSimpleField('Name', "string", "bill heb name")),
+        ("type_id", KnessetDataServiceSimpleField('SubTypeID', "integer", "type id of the bill")),
+        ("type_description",
+         KnessetDataServiceSimpleField('SubTypeDesc', "string", "type description of the bill")),
```

Comment:
the new knesset API have a lot of these fields where you have somethingID and related somethingDesc
worth to create a new KnessetDataServiceField class which will handle it automatically - 
so instead of defining those two fields, you would define only 1 field:
("type", KnessetDataServiceIdDescriptionField("SubType", "type of the bill"))
behind the scenes this field can expose a python tuple of (id, description)